src/components/question/index.js

```diff
@@ -24,11 +25,11 @@ const Question = ({incrementIndex,incrementScore,question, incorrectAnswers, cor
                 </div>
                 <div className={styles.liDiv}>
                     <ol>
-                        {answer.map((answer,i) => <Answers key={i} answers={answer}/>)}
+                        {answer.map((answer,i) => Answers(i, answer))}
```

Comment:
why not returning li directly, no need for the function

Reply:
done